ros2nodl/ros2nodl/_verb/_validate.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2020 Canonical Ltd.
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License Version 3 as published by
+# the Free Software Foundation.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import argparse
+from pathlib import Path
+import pprint
+import shutil
+import sys
+
+from argcomplete.completers import FilesCompleter
+import nodl
+from nodl._index import _FILE_EXTENSION
+from ros2cli.verb import VerbExtension
+
+
+class _ValidateVerb(VerbExtension):
+    """Validate NoDL XML documents."""
+
+    def add_arguments(self, parser: argparse.ArgumentParser, cli_name: None = None):
+        # Ignoring type because of https://github.com/python/typeshed/issues/1878
+        parser.add_argument(  # type: ignore
+            'executables',
+            nargs='*',
+            default=[],
+            metavar='executable',
+            help='Specific .nodl.xml file(s) to validate.',
```

Comment:
Would it make sense to use `_FILE_EXTENSION` here as well?

Reply:
fixed in 3955cc9